### DISCUSSION TITLE
ScriptReference/Graphics.ConvertTexture

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#220] [Notes]
# ScriptReference/Graphics.ConvertTexture

https://docs.unity3d.com/ScriptReference/Graphics.ConvertTexture.html

**Comment by mvi** [comment]
There are multiple methods in SystemInfo to check if a target format is supported as a RenderTexture, such as [`IsFormatSupported`](https://docs.unity3d.com/ScriptReference/SystemInfo.IsFormatSupported.html) with FormatUsage.Render supplied and [`SupportsRenderTextureFormat`](https://docs.unity3d.com/ScriptReference/SystemInfo.SupportsRenderTextureFormat.html).
